lib/basket.rb

```diff
@@ -0,0 +1,23 @@
+class Basket
+  attr_reader :summary
+  attr_accessor :items
+
+  def initialize
+    @items = []
+  end
+
+  def summary(items = unique_items)
+    summary = []
+    count = 0
+    items.each do |item|
+      @items.each { |dish| count += 1 if dish == item }
+      summary.push("#{item.keys[0]} x#{count} = £#{item[item.keys[0]] * count}")
+    end
+    @summary = summary
```

Comment:
No need to store `@summary`.  You can make `summary` return it.  Any code that needs the summary can call `summary`.